components/mpas-ocean/src/shared/mpas_ocn_vel_tidal_potential.F

```diff
@@ -434,9 +430,6 @@ subroutine ocn_vel_tidal_potential_init(domain,err)!{{{
             constituentList(nTidalConstituents)%constituent = 'P1'
          end if 
 
-         ! set point value in forcing pool just in case
-         nTidalPotentialConstituents = nTidalConstituents
```

Comment:
@sbrus89 please check if I can remove `nTidalPotentialConstituents`. It looks like you added it to record information during development. This would never change, so I suspect you don't need it anymore. The gnu compiler seems to think you don't need it either.